PrimeOptions.py

```diff
@@ -320,6 +320,30 @@ class ProgressiveBeamUpgrades(Toggle):
 
 # COSMETIC OPTIONS
 
+class ProgressionPickupModel(Choice):
+    """Choose a model for Progression item pickups"""
+    default = "Cog"
+    option_metroid = "Metroid"
+    option_zoomer= "Zoomer"
+    option_cog = "Cog"
+    option_gamecube = "GameCube"
+
+class UsefulPickupModel(Choice):
+    """Choose a model for Useful or Trap item pickups"""
+    default = "Zoomer"
+    option_metroid = "Metroid"
+    option_zoomer= "Zoomer"
+    option_cog = "Cog"
+    option_gamecube = "GameCube"

```

Comment:
I'd like to see separate Useful and Trap options to give the player the choice to make trap pickups look like progression. (and IMO trap should take priority over useful)